Documentation/Signoffs/Security_Locks.md

```diff
@@ -0,0 +1,58 @@
+# Security & Locks
+
+# Subsystem Function:
+This subsystem's role in our design is to secure both the stored devices and internal components of the machine. The main contributor to the security system will be the cabinet doors holding the devices. These cabinet door require digitally controlled locks, allowing access to verified users, while locking out those who would attempt to forcefully steal from the machine.
+
+
+# Constraints
```

Comment:
Digital is not enough to go on for a design. What are the communication constraints?

Reply:
Previous comments still need addressing.